src/api-mocks/submissions.js

```diff
@@ -143,3 +143,15 @@ export const mockSubmissionPaymentStartGet = rest.post(
     return res(ctx.json({data: {method: 'get', action: 'https://example.com'}}));
   }
 );
+
+export const mockGetAnalyticsToolConfig = (overrides = {}) =>
```

Comment:
this doesn't seem directly related to `submissions`, so I'd put it in a separate module `analytics-config.js` instead :)